.github/workflows/tests.yml

```
@@ -0,0 +1,36 @@
+name: Test schemas for each language
+
+on:
+  push:
+    branches:
+      - main
+  merge_group:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  python-tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        schema:
+          - pattern-escaped-slash-schema.json
+          - pattern-unescaped-slash-schema.json
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

I think we need to use v5 now:
```suggestion
        uses: actions/setup-python@v5

---

Done